fontra2fontir/src/fontra.rs

```diff
@@ -418,7 +436,10 @@ mod tests {
         let glyph = read_test_glyph("minimal.fontra", ".notdef");
         assert_eq!(GlyphName::new(".notdef"), glyph.name, "{glyph:#?}");
         assert_eq!(
-            1000.0, glyph.layers["foreground"].glyph.x_advance,
+            1000.0,
+            glyph.layers[&LayerName("foreground".into())]
```

Comment:
this is the bit that feels particularly bad: where previously we could use a `&str` as the key here (because `String` implements `Borrow<str>` now we need to allocate and be more verbose, in a situation where confusion seems unlikely.

Reply:
Updating to alias.